From 1ccfc1224b41ec56837c95066813ee4f4b3e854b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ton=C4=87i=20Gali=C4=87?= <tuxified@gmail.com>
Date: Mon, 28 Nov 2022 21:15:25 +0100
Subject: [PATCH] hwdb: Add support for Elgato Stream Pedal (#25550)

This adds the [Elgato Stream Pedal](https://www.elgato.com/en/stream-deck-pedal)
as part of the supported family

(cherry picked from commit bcad98ee4d0be0245fba159e2531ef170fbb6a8b)

Related: RHEL-5950
---
 hwdb.d/70-av-production.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/70-av-production.hwdb b/hwdb.d/70-av-production.hwdb
index 5df128f07e..f89f26eb6f 100644
--- a/hwdb.d/70-av-production.hwdb
+++ b/hwdb.d/70-av-production.hwdb
@@ -52,6 +52,10 @@ usb:v0FD9p006D*
 usb:v0FD9p0080*
  ID_AV_PRODUCTION_CONTROLLER=1
 
+# Stream Deck Pedal
+usb:v0FD9p0086*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
 #############################
 # Hercules (Guillemot Corp)
 #############################
